e2e/utils.go

```diff
@@ -136,6 +136,31 @@ func execCommandInPod(f *framework.Framework, c, ns string, opt *metav1.ListOpti
 	return stdOut
 }
 
+func execCommandInToolPod(f *framework.Framework, c, ns string, opt *metav1.ListOptions) string {
```

Comment:
@poornimag use `execCommandInPod` function.

Reply:
Using that fails the command "ceph-fuse" i think though it is successful the stderr is still present. Hence copied it to another function and got rid of check for err and stderr empty. Since it is short lived didn't combine the functions.